src/components/pages/Details.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+import { BiArrowBack } from 'react-icons/bi';
+import { FaBitcoin } from 'react-icons/fa';
+
+const Details = () => {
```

Comment:
- [ ] Kindly use a lighter and darker variation of the color you are using instead of alternating between it and white. 🤓️
- [ ] Please also align the contents of the details as shown in the design. The contents should be aligned to the left and right sides while maintaining as much space as possible in the center. I think you can round the numbers to some decimal points if they are taking up much space and making it harder to style. 🥸️

| Your design                                                             | Template                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://github.com/katfogy/react-capstone-web-metric-app/assets/17243325/e00a472e-bce6-463f-891e-57704d911cc3" width="300">|<img src="https://github.com/katfogy/react-capstone-web-metric-app/assets/17243325/5dbde54a-05a6-4c84-863a-1a707fe61950" width="300">|